### DISCUSSION TITLE
Chinese Terminology

### DISCUSSION THREAD

**Discussion by dreamerlin** [open-mmlab/mmaction2#708] [Design]
English | 中文
-- | --
checkpoint | 模型权重文件
backbone | 主干网络
cls_head | 分类头
pipeline | 流水线
hook | 钩子
finetune | 微调
register | 注册器
tensor | 张量
benchmark | 基准测试
recognizer | 动作识别器
detector | 时空动作检测器
localizer | 时序动作检测器
annotation | 标注
training schedule | 训练策略
scheduler | 调度器
schedule | 调整

**Comment by dreamerlin** [comment]
duplicate due to https://github.com/open-mmlab/mmaction2/issues/713
@innerlee

**Reply by innerlee** [reply]
@congee524  Let's put it here

**Comment by hellock** [comment]
A suggestion for checkpoint: 模型权重文件

**Reply by innerlee** [reply]
both are ok to me

**Reply by congee524** [reply]
vote for 模型权重文件 or 模型参数文件
